app/src/test/java/ar/nablab/mimerendero/dtos/PlaceTest.kt

```diff
@@ -0,0 +1,30 @@
+package ar.nablab.mimerendero.dtos
```

Comment:
Tests should pass on Circle-CI, maybe we can look into it.
 [CIRCLE](https://app.circleci.com/pipelines/github/nablab-opencode/lamp)
apparently right now the service is on maintenance...I'll check tomorrow.